partcad-cli/src/partcad_cli/click/commands/adhoc/convert.py

```
@@ -0,0 +1,80 @@
+        logging.error("Cannot infer output type. Please specify --output explicitly.")
+        raise click.Abort()
+
+    if not output_filename:
+        output_filename = Path(input_filename).stem + f".{output_type}"
+
+    # Perform conversion
+    try:
+        logging.info(f"Converting {input_filename} ({input_type}) to {output_filename} ({output_type})...")
+        convert_cad_file(input_filename, input_type, output_filename, output_type)
+        logging.info(f"Conversion complete: {output_filename}")
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)